src/styles/card.css

```diff
@@ -0,0 +1,98 @@
+/* --background-color: #fff;
+--border-color: #f3f3f3;
+--title-color: #060b35;
+--subtitle-color: #616070;
+--color-for-link: #00acdc; */
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 166px;
+  background-color: var(--white-color);
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+}
+
+.card__image {
+  /* margin-bottom: 40px; */
+  width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-top: 40px;
+  margin-left: 1px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--title-color);
+  /* margin-bottom: 4px; */
+}
+
+.card__subtitle {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--subtitle-color);
+  margin-top: 4px;
+  /* margin-bottom: 16px; */
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--title-color);
+}
+
+.card__pricing {
+  display: flex;
+  justify-content: space-between;
+  /* margin-bottom: 16px; */
+  margin-top: 24px;
+}
+
+.card__text {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--subtitle-color);
+}
+
+.card__price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--title-color);
+}
+
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--color-for-link);
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: var(--white-color);
+  text-decoration: none;
+  border: 1px solid var(--color-for-link);
+  padding: 11px 69px;
```

Comment:
It is not necessary to transfer everything that is in the figma, some sizes should be dynamic, 12 pixels from top to bottom, and automatic on the left and right
```suggestion
  padding: 12px 0;
```
![image](https://user-images.githubusercontent.com/70009205/217291497-43aacaa2-f46f-4b5e-9363-11c09b436b7b.png)